### ISSUE TITLE
[Feature Request]: Easy access to problem supporting evidence within SCT environment; linkage to HIS and planning reach within SCT

### ISSUE THREAD

**Issue by marko-nedzbala** [USACE/chart-docs#28]
### Background

View, display and/or report CBRA zones  closest to and/or within study reach or planning reach boundaries

### User Story

Make PDT aware of  CBRA zones so they can be factored into plan formulation constraints within SCT

### Purpose

Identify Risk: develop & substantiate problems & opportunities / Define study objectives & constraints

### Workflow

Roles include:
- PTL

**Comment by trietmnj** [comment]
Please provide more information on the "CBRA zones". First step is to note what the acronym stands for.

**Comment by trietmnj** [comment]
@marko-nedzbala do you mind reaching out to whoever made this comment to get more details?

**Comment by marko-nedzbala** [comment]
I will research it and hopefully get an answer.

**Comment by trietmnj** [comment]
CBRA = Coastal Barrier Resources Act. Fish and Wildlife Services might be the maintainer. 

Just heard this - might be a good place to start.

**Comment by KHodgens** [comment]
https://fwsprimary.wim.usgs.gov/cbrs-mapper-v2/

**Comment by KHodgens** [comment]
My understanding of CBRA zones is that the fed gov is limited with respect to actions involving sediment dredging and placement.  There are regional FWS offices that provide rulings on USACE activities in CBRA zones and they have varied regionally where SAJ was able to do something but SAC (or SAW) was denied for a very similar effort

**Comment by KHodgens** [comment]
Yes I believe the dataset covers a significant portion of the US coastline but my experience is limited to SAJ AOR
